packages/server/src/prisma.service.ts

```diff
@@ -0,0 +1,13 @@
+import { INestApplication, Injectable, OnModuleInit } from '@nestjs/common';
```

Comment:
Maybe move this into it's own prisma folder, since it a file we don't need to interact with on a day to day basis